SpaceX/Extensions/Extension + UIViewController.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Extension + UIViewController.swift
+//  SpaceX
+//
+//  Created by Julia on 26.07.2022.
+//
+
+import UIKit
+
+extension UIViewController {
+    func startAnimationLoaderView() -> ActivityIndicator {
+        let rocketsView = ActivityIndicator(frame: CGRect(x: 0, y: 0, width: 0, height: 0))
+        view.addSubview(rocketsView)
+        rocketsView.accessibilityIdentifier = "indicatorView"
```

Comment:
Your app does not support accesibility thats why you can remove it` rocketsView.accessibilityIdentifier = "indicatorView"`